.github/workflows/app_build.yml

```diff
@@ -451,52 +445,46 @@ jobs:
-
-    # Note: JM 2020-07-22 This is an example of how to get a config log for a failed conan dependency build (no binary available)
-    # This WILL be handy some day, so leave it here
-    #- name: upload config.log for failed gdbm build
-      #if: ${{ failure() }}
-      #shell: bash
-      #run: |
-        #configlog=$(find ~/.conan/data/gdbm/1.18.1/_/_/build -name config.log)
-        #cat $configlog
-
-    #- name: upload config.log for failed gdbm build
-      #if: ${{ failure() }}
-      #uses: actions/upload-artifact@v4
-      #with:
-          #name: ${{ matrix.os }}-gdbm_config.log
-          #path: ~/.conan/data/gdbm/1.18.1/_/_/build/**/source_subfolder/config.log
-
-    - name: Build (Unix)
-      working-directory: ./build
-      if: runner.os != 'Windows'
-      shell: bash
-      # Execute the build.  You can specify a specific target with "--target <NAME>"
-      run: |
-        set -x
-        ninja
-        ninja package || ( echo "Package Step failed" && cpack --debug --verbose --config CPackConfig.cmake )
+        if [ "$RUNNER_OS" == "macOS" ]; then
+          # Avoid "builtin __has_nothrow_assign is deprecated; use __is_nothrow_assignable instead" in boost/1.78 with recent clang
+          conan install . --output-folder=./build --build=missing -c tools.cmake.cmaketoolchain:generator=Ninja -s compiler.cppstd=20 -s build_type=Release -c tools.build:cxxflags="['-D_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION']"
+        else
+          conan install . --output-folder=./build --build=missing -c tools.cmake.cmaketoolchain:generator=Ninja -s compiler.cppstd=20 -s build_type=Release
+        fi
+
+        if [ "${{ matrix.SELF_HOSTED }}" == "true" ]; then
+          # TODO: temp... this is unfortunate but the OS SDK arm64 rc2 package has a different sha when extracted...
+          # OpenStudio-3.8.0-rc2+af997f4fef-Darwin-x86_64.tar.gz on github, but when extracted it's OpenStudio-3.8.0-rc2+06d9d975e1-Darwin-arm64
+          cmake --preset conan-release -DQT_INSTALL_DIR:PATH=${{ env.QT_INSTALL_DIR }} \
+              -DBUILD_DOCUMENTATION:BOOL=${{ env.BUILD_DOCUMENTATION }} \
+              -DBUILD_PACKAGE:BOOL=${{ env.BUILD_PACKAGE }} \
+              -DANALYTICS_API_SECRET:STRING=${{ secrets.ANALYTICS_API_SECRET }} \
+              -DANALYTICS_MEASUREMENT_ID:STRING=${{ secrets.ANALYTICS_MEASUREMENT_ID }} || true
+
+          ls $(pwd)/build/OpenStudio-3.8.0 || true
+          OS_DIR=$(find "$(pwd)/build/OpenStudio-3.8.0" -name openstudioConfig.cmake)
+          OS_DIR=$(dirname $OS_DIR)
+          cmake --preset conan-release -DQT_INSTALL_DIR:PATH=${{ env.QT_INSTALL_DIR }} \
+              -DBUILD_DOCUMENTATION:BOOL=${{ env.BUILD_DOCUMENTATION }} \
+              -DBUILD_PACKAGE:BOOL=${{ env.BUILD_PACKAGE }} \
+              -DANALYTICS_API_SECRET:STRING=${{ secrets.ANALYTICS_API_SECRET }} \
+              -DANALYTICS_MEASUREMENT_ID:STRING=${{ secrets.ANALYTICS_MEASUREMENT_ID }} -Dopenstudio_DIR:PATH=$OS_DIR
+        fi
```

Comment:
Need to remove that block soon. OS SDK 3.8.0-RC2 has a different extracted folder (different SHA) than the github hosted tar.gz and that pwned me real bad.